execution/db-bootstrapper/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "db-bootstrapper"
+version = "0.1.0"
+authors = ["Libra Association <opensource@libra.org>"]
+description = "Libra DB-Bootstrapper."
+repository = "https://github.com/libra/libra"
+homepage = "https://libra.org"
+license = "Apache-2.0"
+publish = false
+edition = "2018"
+
+[dependencies]
+anyhow = "1.0"
+structopt = "0.3.13"
+
+executor = { path = "../executor", version = "0.1.0" }
+lcs = { path = "../../common/lcs", version = "0.1.0", package = "libra-canonical-serialization" }
+libradb = { path = "../../storage/libradb", version = "0.1.0" }
+libra-crypto = { path = "../../crypto/crypto", version = "0.1.0" }
+libra-types = { path = "../../types", version = "0.1.0" }
+libra-vm = { path = "../../language/libra-vm", version = "0.1.0" }
+libra-workspace-hack = { path = "../../common/workspace-hack", version = "0.1.0" }
```

Comment:
why we need this?

Reply:
I think it's supposed to be in every crate (to improve build caching)?